### DISCUSSION TITLE
can't install copyq

### DISCUSSION THREAD

**Discussion by arky217** [SpiralLinux/SpiralLinux-project#420] [Q&A]
when i try to install copyq via synaptic, it goes red when i mark it for installation and when i click apply it says 'fix broken packages first'.
when I try to install it via sudo apt-get install copyq, it shows the message 
The following packages have unmet dependencies:
 libqt5qml5 : Depends: qtbase-abi-5-15-8
 libqt5texttospeech5 : Depends: qtbase-abi-5-15-8
 libqt5waylandclient5 : Depends: qtbase-abi-5-15-8
E: Unable to correct problems, you have held broken packages.

and when I try to install qtbase-abi-5-15-8, it gives the message

Package qtbase-abi-5-15-8 is a virtual package provided by:
  libqt5core5a 5.15.8+dfsg-11 [Not candidate version]

E: Package 'qtbase-abi-5-15-8' has no installation candidate

I've never had a problem before installing copyq on other distros.
What could the problem be ?

**Comment by geckolinux** [comment]
Hi there, are you trying to install the version from the Debian Stable repository or from a DEB package?

**Reply by arky217** [reply]
First using synaptic from the Debian Stable repository.
Then from the terminal using apt-get install.

**Reply by geckolinux** [reply]
I also use `copyq` from Debian Stable on my system. Did you maybe add or remove some repository sources to/from your system?

**Comment by arky217** [comment]
I did add the sid repository just long enough to install Nomacs, but then I removed it again right away.
Did not do any updating during the few minutes the sid repository was added.

Sent with [Proton Mail](https://proton.me/) secure email.

**Comment by arky217** [comment]
Also, when I try to install libqt5qml5 or libqt5texttospeech5 or libqt5waylandclient5, it says that they need
qtbase-abi-5-15-8, but then when I try to install qtbase-abi-5-15-8, it gives the message

Package qtbase-abi-5-15-8 is a virtual package provided by:
libqt5core5a 5.15.8+dfsg-11 [Not candidate version]

E: Package 'qtbase-abi-5-15-8' has no installation candidate

Sent with [Proton Mail](https://proton.me/) secure email.

**Reply by geckolinux** [reply]
It must be an issue with some packages that got upgraded when you installed Nomacs. Mixing different branches of Debian will usually cause issues.

**Comment by arky217** [comment]
Ok, thanks for the replies. I think I'll just do a fresh install and keep my fingers off of the testing and sid repositories.

Sent with [Proton Mail](https://proton.me/) secure email.

**Reply by geckolinux** [reply]
By any chance did you install with the default Btrfs configuration? If so you could do a rollback:
https://github.com/SpiralLinux/SpiralLinux-project/wiki#working-with-btrfs-snapshots-and-rollbacks

**Comment by arky217** [comment]
No, I used ext4. I like to make regular snapshots using Timeshift onto an external SSD, and Btrfs doesn't seem to allow snapshots outside of the system. At least, that's the way it seemed to me, but maybe I'm
overlooking something.

Sent with [Proton Mail](https://proton.me/) secure email.

**Reply by geckolinux** [reply]
I'm pretty sure that you can do that with Btrfs, for example:
https://forum.manjaro.org/t/howto-use-btrfs-send-receive/135808
